src/main/java/uk/ac/ebi/embl/flatfile/writer/FlatFileWriter.java

```diff
@@ -109,16 +111,16 @@ protected void writeBlock(Writer writer, String header,
 	 */
 	protected void writeBlock(Writer writer, String firstLineHeader,
 									 String header, String block) throws IOException {
-		writeBlock(writer, firstLineHeader, header, block, wrapChar,wrapType, header.length(), forceLineBreak, this.optimalLineLength);
+		writeBlock(writer, firstLineHeader, header, block, wrapChar,wrapType, header.length(), forceLineBreak, this.optimalLineLength, this.customMaximumLineLength);
 	}
 
 	public static void writeBlock(Writer writer, String firstLineHeader,
-			String header, String block, WrapChar wrapChar, WrapType wrapType, int headerLength, boolean forceBreak, Integer optimalLineLength) throws IOException {
+			String header, String block, WrapChar wrapChar, WrapType wrapType, int headerLength, boolean forceBreak, Integer optimalLineLength, Integer customMaxLineLength) throws IOException {
```

Comment:
would just call this:
```
maximumLineLength
```
like the
```
optimalLineLength
```
One of these is not more custom than the other.